nerdlets/shared/container-charts.js

```diff
@@ -22,15 +22,15 @@ function summarizeFacets(data) {
 }
 
 function getNrql({ select, timeRange, containerId }) {
-  return `SELECT ${select} FROM ProcessSample 
-    WHERE containerId='${containerId}' FACET processId 
+  return `SELECT ${select} FROM ProcessSample
+    WHERE containerId='${containerId}' FACET processId
     TIMESERIES LIMIT MAX ${timeRange}`;
 }
 
 function Chart({ account, containerId, select, timeRange }) {
   const nrql = getNrql({ select, timeRange, containerId });
   return (
-    <NrqlQuery accountId={account.id} query={nrql}>
+    <NrqlQuery accountIds={account.id} query={nrql}>
```

Comment:
`accountIds` is now an array
```suggestion
    <NrqlQuery accountIds={[account.id]} query={nrql}>
```